File: Lab01/C/Euklidesa.c
#include <stdio.h>

int NWD(int a, int b){
    while (b > 0){
        int c = a % b;
        a = b;
        b = c;
    }
    return a;
}

int main(){
    int a,b,c;

    printf("Podaj pierwsza liczbe: ");
    scanf("%d", &a);
    printf("Podaj druga liczbe: ");
    scanf("%d", &b);
    c = NWD(a,b);
    printf("Największy wspolny dzielnik to %d\n", c);
    
    return 0;
}

File: Lab02/Python/Euklidesa.py

def Euklides(a,b):
    while b:
        c = a % b
        a = b
        b = c
    return a
if __name__ == "__main__":
    a = int(input("Podaj pierwsza liczbe: "))
    b = int(input("Podaj druga liczbe: "))
    print(f"Największy wspólny dzielnik: {Euklides(a,b)}")